ui/app/routes/policies.js

```diff
@@ -0,0 +1,25 @@
+import Route from '@ember/routing/route';
+import withForbiddenState from 'nomad-ui/mixins/with-forbidden-state';
+import WithModelErrorHandling from 'nomad-ui/mixins/with-model-error-handling';
+import { inject as service } from '@ember/service';
+
+export default class PoliciesRoute extends Route.extend(
+  withForbiddenState,
+  WithModelErrorHandling
+) {
+  @service can;
+  @service store;
+  @service router;
+
+  beforeModel() {
+    if (this.can.cannot('list policies')) {
+      this.router.transitionTo('/jobs');
+    }
+  }
+
+  async model() {
+    const policies = await this.store.query('policy', { reload: true });
+    const tokens = await this.store.query('token', { reload: true });
```

Comment:
`nit`: You could parallelize these promises so that you're not waiting for `policy` to fetch to begin fetching `tokens`.

```
async model() {
 const [policies, tokens] = Promise.all(queryPolicies, queryTokens);
 return {policies, tokens};
}
```

Reply:
Good suggestion; going to use Ember's RSVP.hash to do this more idiomatically.